R/summarize.R

```diff
@@ -4,7 +4,7 @@
 summarise_.CrunchDataset <- function (.data, ..., .dots) {
     dots <- all_dots(.dots, ..., all_named = TRUE)
     fmla <- dots_to_formula(dots, groups(.data))
-    out <- as_tibble(crtabs(fmla, data=.data))
+    out <- as_tibble(crtabs(fmla, data=.data), show_metadata = FALSE)
```

Comment:
It looks like this is the only reason for this argument. I wonder whether we need this argument at all. Why do you think it should be `FALSE` here?

Reply:
Mostly it was to not change the summarize behaviour with the updates to `as.tibble.` I think the current behaviour of summarize is that missing values are suppressed and unweighted counts aren't included and I think that that behaviour makes sense because `dplyr::summarize` only returns the summary stats that are included as arguments to summarize. So basically when I did `ds %>% summarize(mean = mean(mpg))` my intuition was that it should return the grouping variables and the `mean` column and not include a bunch of extra metadata.